terraform/codebase-pipelines/buildspec-images.yml

```diff
@@ -38,8 +38,39 @@ phases:
           BUILD_ID_PREFIX=$(echo $CODEBUILD_BUILD_ID | cut -d':' -f1)
           echo "BUILD_ID_PREFIX - ${BUILD_ID_PREFIX}"
 
-          echo -e "\nInstalling dependencies"
-          pip install dbt-platform-helper
+          # Install Platform Helper
+          ## Clone platform-tools
+          cd "${CODEBUILD_SRC_DIR}"
+          codestar_connection_id=$(echo "${CODESTAR_CONNECTION_ARN##*/}")
+          export GIT_CLONE_BASE_URL="https://codestar-connections.${AWS_REGION}.amazonaws.com/git-http/${AWS_ACCOUNT_ID}/${AWS_REGION}/$codestar_connection_id/uktrade"
+          git config --global credential.helper '!aws codecommit credential-helper $@'
+          git config --global credential.UseHttpPath true
+          git config --global advice.detachedHead false
+          git clone "${GIT_CLONE_BASE_URL}/platform-tools.git" --branch "${PLATFORM_HELPER_VERSION}" --depth 1 platform-tools
+          cd "${CODEBUILD_SRC_DIR}/platform-tools"
+          echo -e "\nCurrent platform-tools branch/commit: $(git rev-parse --abbrev-ref HEAD)/$(git rev-parse HEAD)"
+          ## Set up python venv
+          echo -e "Set up venv"
+          rm -rf venv
+          python -m venv --copies venv
+          echo -e "\nUpdating paths"
+          echo -e "\nActivating virtual environment"
+          source venv/bin/activate
+          ## yq for yaml processing
+          pip install yq --quiet
+          ## Build platform-helper
+          echo -e "\n\n### Build and install platform-helper cloned from ${PLATFORM_HELPER_VERSION}\n"
+          echo -e "Installing dependencies"
+          pip install poetry
+          poetry install
+          echo -e "\nBuild platform-helper\n"
+          poetry build --no-interaction --format sdist --no-ansi --local-version ${PLATFORM_HELPER_VERSION}
+          echo -e "\nInstall platform-helper\n"
+          pip install "dist/$(ls -t1 dist | head -1)"
```

Comment:
My goodness, my eyes bleed at this line.

Reply:
I think just hardcoding `dist/platform-helper` would be simpler? OR am I confused?